data_handler/handlers/liquidable_debt/debt_handlers.py

```diff
@@ -275,3 +275,94 @@ def calculate_liquidable_debt(self, protocol_name: str = None) -> list:
+    """
+
+    def __init__(
+            self,
+            loan_state_class: Type[State],
+            loan_entity_class: Type[LoanEntity],
+    ):
+        super().__init__()
+        self.state_class = loan_state_class
+        self.loan_entity_class = loan_entity_class
+
+    def initialize_loan_entities(self, state: State, data: dict = None):
+        """
+        Initializes the loan entities in a state instance.
+        :param state: State
+        :param data: dict
+        :return: None
+        """
+
+        for instance in data:
+            debt_category = self.db_connector.get_last_hashstack_loan_state(instance.user).debt_category
+
+            loan_entity = self.loan_entity_class(
+                user=instance.user,
+                debt_category=debt_category
+            )
+
+            loan_entity.debt = TokenValues(values=instance.debt)
+            loan_entity.collateral = TokenValues(values=instance.collateral)
+
+            state.loan_entities.update(
+                {
+                    instance.user: loan_entity,
+                }
+            )
+
+    def calculate_liquidable_debt(self, protocol_name: str = None) -> list:
+        """
+        Calculates liquidable debt based on data provided and updates an existing data.
+        Data to calculate liquidable debt for:
+        :param protocol_name: str
+        :return: A dictionary of the ready liquidable debt data.
+        """
+        data, interest_rate_models = self.fetch_data(protocol_name=protocol_name)
+        state = self.state_class()
+        self.initialize_loan_entities(state=state, data=data)
+
+        # Set up collateral and debt interest rate models
+        state.collateral_interest_rate_models = TokenValues(
+            values=interest_rate_models.collateral
+        )
+        state.debt_interest_rate_models = TokenValues(
+            values=interest_rate_models.debt
+        )
+
+        current_prices = Prices()
+        asyncio.run(current_prices.get_lp_token_prices())
+
+        hypothetical_collateral_token_prices = self.get_prices_range(
+            collateral_token_name="STRK",
+            current_price=current_prices.prices.values["STRK"]
+        )
+
+        result_data = list()
+        # Go through first hypothetical prices and then through the debts
+        for hypothetical_price in hypothetical_collateral_token_prices:
+            liquidable_debt = state.compute_liquidable_debt_at_price(
+                prices=TokenValues(values=current_prices.prices.values),
+                collateral_token="STRK",
+                collateral_token_price=hypothetical_price,
+                debt_token="USDC",
+            )
+
+            if liquidable_debt > Decimal("0"):
+                result_data.append({
+                    LIQUIDABLE_DEBT_FIELD_NAME: liquidable_debt,
+                    PRICE_FIELD_NAME: hypothetical_price,
+                    COLLATERAL_FIELD_NAME: "STRK",
```

Comment:
What is the magic variable `STRK` if we handle some specific case, please provide with extra comments.
From first glance it's not clear